Assets/IATK/Scripts/Demo/ReplicatorTest.cs

```diff
@@ -0,0 +1,402 @@
+﻿//ReplicatorTest
+//Philipp Fleck, ICG @ TuGraz, philipp.fleck@icg.tugraz.at
+//20210708, initial working version
+
+//#define USE_MQTT
+
+using IATK;
+using System;
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+namespace IATKTest
+{
+    public class ReplicatorTest : MonoBehaviour
+    {
+        Replicator repl = null;
+        GameObject visGo = null;
+        Visualisation vis = null;
+        RealtimeDataSource rtds = null;
+
+        bool isVisReady = false;
+
+        void Start()
+        {
+            StartCoroutine(DelayedInit());
+        }
+
+        IEnumerator DelayedInit()
+        {
+            yield return new WaitForSeconds(2f);
+
+#if UNITY_EDITOR
+            yield return new WaitForSeconds(0.5f);
+            CreateWithRTDataSource();
+#endif
+
+            yield return new WaitForSeconds(3f);
+            repl = FindObjectOfType<IATK.Replicator>();
+
+            if (repl == null)
+            {
+                Debug.Log("ReplicatorTest repl not found, adding new");
+                repl = gameObject.AddComponent<IATK.Replicator>();
+            }
+            Debug.Log("ReplicatorTest repl => " + repl + ", " + repl.gameObject);
+
+            if (repl != null)
```

Comment:
No need to check if 'repl != null' as you set repl above it, so repl will always be not null here